calculator.c

```diff
@@ -0,0 +1,117 @@
+#include<stdio.h>
+#include<string.h>
+#include<ctype.h>
+#include<stdlib.h>
+//store number in stack
+int numStack[30];
+int topnum=-1;
+//store operators in stack
+char opStack[20];
+int topop=-1;
```

Comment:
Why do we need these variables as global?

Reply:
Since these variables are used in more than one function, So it has to be declared it globally

Reply:
Why should we avoid usage of Global Variables?

- Hard to Debug: Changes can come from anywhere, making debugging difficult.
- Name Conflicts: Risk of overwriting variables in large codebases.
- Poor Testing: Shared state complicates unit testing.
- Less Reusable: Functions relying on globals can't be reused easily.

**Alternatives**: look for the better alternatives and use them here